instrumentation/grpc-1.6/testing/src/main/groovy/io/opentelemetry/instrumentation/grpc/v1_6/AbstractGrpcTest.groovy

```diff
@@ -143,6 +143,124 @@ abstract class AbstractGrpcTest extends InstrumentationSpecification {
     paramName << ["some name", "some other name"]
   }
 
+  def "test callback"() {
```

Comment:
Can we add a test using GreeterFutureStub too? It should work but will test the more common usage.